zone/spawngroup.cpp

```diff
@@ -191,7 +191,8 @@ bool ZoneDatabase::LoadSpawnGroups(const char *zone_name, uint16 version, SpawnG
 				WHERE
 				spawn2.spawngroupID = spawngroup.ID
 				AND
-				spawn2.version = {} and zone = '{}'
+				(spawn2.version = {} OR version = -1)
```

Comment:
For future note; not prefixing the table in a join can throw an error in the query parser depending on how the query is built

I am making an assumption this was tested

Reply:
Yeah I tested it.. Looks like in this case its okay, but will keep in mind in the future

![91fa7a92e0387e50cdb47dbf10100198](https://user-images.githubusercontent.com/4369261/111266354-d238ec80-8600-11eb-86d1-0c56b59ada92.png)

Reply:
my guess is it is not throwing an error because there is no version field in the spawngroup table?